[BUG] - Failing Cypress tests in main

### Describe the bug

The cypress tests are failing in main:
https://github.com/nebari-dev/nebari/actions/runs/9067624675/job/24914178045#step:17:159

### Expected behavior

Testing passing

### OS and architecture in which you are running Nebari

Linux

### How to Reproduce the problem?

Run integration tests in the CI

### Command output

_No response_

### Versions and dependencies used.

_No response_

### Compute environment

None

### Integrations

_No response_

### Anything else?

_No response_

>        Check Nebari login and start JupyterLab:
>      AssertionError: Timed out retrying after 4000ms: Expected to find element: `input.btn.btn-jupyter`, but never found it.

It is getting stuck on server selection screen:

![image](https://github.com/nebari-dev/nebari/assets/5832902/7b39c65b-f48c-4a43-abf3-916ad42a1c80)

Sounds like this could be related to Bootstrap 5 changes in JupyterHub 5. Maybe changing it to `button.btn.btn-jupyter` would be sufficient.

--- a/tests/tests_e2e/cypress/integration/main.js
+++ b/tests/tests_e2e/cypress/integration/main.js
@@ -52,7 +52,7 @@ describe('First Test', () => {
         cy.get('h1')
         .should('contain', 'Server Options');
 
-      cy.get('input.btn.btn-jupyter')
+      cy.get('button.btn.btn-jupyter')
         .should('have.attr', 'value', 'Start').click();
 
       // Minimal check that JupyterLab has opened

--- a/tests/tests_e2e/cypress/integration/main.js
+++ b/tests/tests_e2e/cypress/integration/main.js
@@ -53,7 +53,7 @@ describe('First Test', () => {
         .should('contain', 'Server Options');
 
       cy.get('button.btn.btn-jupyter')
-        .should('have.attr', 'value', 'Start').click();
+        .should('contain', 'Start').click();
 
       // Minimal check that JupyterLab has opened
       cy.get('div#jp-MainLogo', { timeout: 60000 }).should('exist').wait(4000);

